deploy/helm/sumologic/templates/deployment.yaml

```diff
@@ -33,6 +33,8 @@ spec:
       - name: config-volume
         configMap:
           name: {{ template "sumologic.fullname" . }}
+      securityContext:
+        fsGroup: {{ .Values.sumologic.fluentd.securityContext.fsGroup }}
```

Comment:
heads up @samjsong , this needs to be updated in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/330

Reply:
there's quite some changes after renaming `deployment` to `statefulset` in different places, @samjsong maybe I can work on them after this PR is merged and contribute to your PR for any `values.yaml` changes needed to avoid more conflicts

Reply:
that would be great, thanks @maimaisie ! Looks like this PR and #339 should be in a minor release before #330 so that should work